astropy/visualization/scripts/fits2bitmap.py

```diff
@@ -71,12 +70,24 @@ def fits2bitmap(filename, ext=0, out_fn=None, scale='linear',
     import matplotlib.cm as cm
     import matplotlib.image as mimg
 
-    hdulist = fits.open(filename)
-    image = hdulist[ext].data
-    hdulist.close()
+    try:
+        ext = int(ext)
+    except ValueError:
+        pass
+
+    # NOTE: getdata raises IndexError if no data is found
```

Comment:
Ah, okay.  In that case this should catch the possible `IndexError`.  Being a command-line script this should never crash--any known exceptions should be caught and handled (this is one of the few things I miss about Java--when written properly anyways it's easy to check what exceptions a method might raise).